alembic/versions/2022_10_13_0054-71b291295ae1_update_schema_types.py

```diff
@@ -0,0 +1,203 @@
+"""Update schema types
```

Comment:
I just ran `alembic revision --autogenerate` without any changes to the model. All the changes you see here are because of the upgrade in `sqlmodel`. This way, when we add a new migration (and I'm planning to because of https://github.com/DataJunction/dj/issues/193) we'll see only relevant changes.